source/renderer/app/domains/Wallet.js

```diff
@@ -3,68 +3,53 @@ import { pick } from 'lodash';
 import { observable, computed, action } from 'mobx';
 import BigNumber from 'bignumber.js';
 import type {
-  WalletAssuranceLevel,
-  WalletAssuranceMode,
   WalletSyncState,
-  SyncStateTag,
+  SyncStateStatus,
+  DelegationStatus,
 } from '../api/wallets/types';
 import type { StakePool } from '../api/staking/types';
 
-export const WalletAssuranceModeOptions: {
-  NORMAL: WalletAssuranceLevel,
-  STRICT: WalletAssuranceLevel,
-} = {
-  NORMAL: 'normal',
-  STRICT: 'strict',
-};
-
-export const WalletSyncStateTags: {
-  RESTORING: SyncStateTag,
-  SYNCED: SyncStateTag,
+export const WalletSyncStateStatuses: {
+  RESTORING: SyncStateStatus,
+  SYNCED: SyncStateStatus,
 } = {
   RESTORING: 'restoring',
-  SYNCED: 'synced',
+  SYNCED: 'ready',
```

Comment:
Should it stay SYNCED: 'synced' in order to keep consistent with the naming, because if we are change it to value 'ready', then the param name should be also changed to READY.

Reply:
@DeeJayElly I agree